docs/application/native/tutorials/details/tizen-apis.md

```diff
@@ -50,20 +50,20 @@ APIs to use.
 | [Base](../../../../org.tizen.native.mobile.apireference/group__CAPI__BASE__FRAMEWORK.html) | [Common  Error](../../../../org.tizen.native.mobile.apireference/group__CAPI__COMMON__ERROR.html) | Provides  error codes that are common for the whole Tizen API. | The  Base API contains submodule APIs for internationalization support and  contains various open source libraries.<br>Guides:<br> - [Data Storage and Management](../../guides/data/data-cover.md)<br> - [Internationalization](../../guides/internationalization/internationalization-cover.md) |
 | [Base](../../../../org.tizen.native.mobile.apireference/group__CAPI__BASE__FRAMEWORK.html) | [Glib](../../../../org.tizen.native.mobile.apireference/group__OPENSRC__GLIB__FRAMEWORK.html) | Provides  low-level libraries and advanced data structures, such as linked lists and  hash tables, for use in application development. | The  Base API contains submodule APIs for internationalization support and  contains various open source libraries.<br>Guides:<br> - [Data Storage and Management](../../guides/data/data-cover.md)<br> - [Internationalization](../../guides/internationalization/internationalization-cover.md) |
 | [Base](../../../../org.tizen.native.mobile.apireference/group__CAPI__BASE__FRAMEWORK.html) | [Glibc](../../../../org.tizen.native.mobile.apireference/group__OPENSRC__GLIBC__FRAMEWORK.html) | Provides  a standard C library.          | The  Base API contains submodule APIs for internationalization support and  contains various open source libraries.<br>Guides:<br> - [Data Storage and Management](../../guides/data/data-cover.md)<br> - [Internationalization](../../guides/internationalization/internationalization-cover.md) |
-| [Base](../../../../org.tizen.native.mobile.apireference/group__CAPI__BASE__FRAMEWORK.html) | [LibXML](../../../../org.tizen.native.mobile.apireference/group__OPENSRC__LIBXML__FRAMEWORK.html) | [Used  to parse XML documents, such as the Application Manifest.](../../index.md#appmanifest) | The  Base API contains submodule APIs for internationalization support and contains various open source libraries.<br>Guides:<br> - [Data Storage and Management](../../guides/data/data-cover.md)<br> - [Internationalization](../../guides/internationalization/internationalization-cover.md) |
+| [Base](../../../../org.tizen.native.mobile.apireference/group__CAPI__BASE__FRAMEWORK.html) | [LibXML](../../../../org.tizen.native.mobile.apireference/group__OPENSRC__LIBXML__FRAMEWORK.html) | Used  to parse XML documents, such as the [Application Manifest.](../../index.md#appmanifest) | The  Base API contains submodule APIs for internationalization support and contains various open source libraries.<br>Guides:<br> - [Data Storage and Management](../../guides/data/data-cover.md)<br> - [Internationalization](../../guides/internationalization/internationalization-cover.md) |
```

Comment:
data-cover.md should be changed into overview.md.
internationalization-cover.md should be changed into overview.md

Reply:
All contents were moved to native/guides/index.md so I delete it.